runtime/vm/JFRChunkWriter.hpp

```diff
@@ -422,6 +422,7 @@ class VM_JFRChunkWriter {
 
 			if (dumpCalled) {
 				writeThreadDumpEvent();
+				_constantPoolTypes.loadNativeLibraries(_currentThread);
```

Comment:
Is this not redundant? See lines 188 and 194 of `JFRWriter.hpp`.

Reply:
Yes I forgot to delete this line. Fixed.

Reply:
Opened https://github.com/eclipse-openj9/openj9/pull/21835